### ISSUE TITLE
Protein ID Workflow Issues

### ISSUE THREAD

**Issue by Stortebecker** [Stortebecker/training-material#1]
1. PeakPickerHiRes crashed mit der Meldung:
	`Fatal error: Exit code 2 ()
/data/6/galaxy_db/job_working_directory/001/240/1240851/tool_script.sh:
line 9: syntax 	error near unexpected token '('`
	`/data/6/galaxy_db/job_working_directory/001/240/1240851/tool_script.sh:
line 9: '[ "$CONDA_DEFAULT_ENV" = "`

2. Mgf-Output vom FileConverter wird nicht automatisch als mgf erkannt,
das manuelle Ändern dauert recht lang (mehrere Minuten)

3. Search GUI hat als Default-Setting OMSSA eingestellt, crasht dann
aber, weil OMSSA nicht läuft. Man sollte vielleicht auf unserer Instanz
nur die Suchmaschinen anbieten, die auch laufen. Sprich: OMSSA und Comet
entfernen. Oder wenigstens OMSSA nicht als Default einstellen.

4. Search GUI bastelt sich gerne selbst seine eigene Decoy-Datenbank.
Die wird doch bestimmt in dem SearchGUI Archive gespeichert. Könnte man
die als optionalen zweiten Output anbieten, falls man die später nochmal
braucht? Diese Datenbank-Sache könnte noch problematisch werden, weil PS
den Decoy-Tag an die Uniprot-Accession hängt. Das habe ich bisher noch
nicht so gesehen und ist AFAIK nicht kompatibel mit OpenMS Tools.

**Comment by blankclemens** [comment]
Yes, way better here.

1. Is fixed in theory, now has to be updated on the server.

**Comment by bgruening** [comment]
Today in the evening ... promised.

**Comment by Stortebecker** [comment]
I am not in a hurry, finished my hours for this week :)

**Comment by blankclemens** [comment]
2. Habe automatische Erkennnung des Output Formates aktiviert, scheint soweit gut zu funktionieren.

**Comment by Stortebecker** [comment]
@blankclemens Zu 2: Noch nicht auf dem Freiburger Server geupdatet, oder?

**Comment by Stortebecker** [comment]
sorry

**Comment by Stortebecker** [comment]
FileInfo crasht mit der üblichen Meldung `Warning: Cannot create folder '.OpenMS' in user home directory. Please check your environment! Home directory determined is: /usr/local/galaxy. Warning: Could not determine format of input file '/data/6/galaxy_db/files/001/957/dataset_1957175.da`

**Comment by blankclemens** [comment]
@Stortebecker Anscheinend nicht - ich hab das nicht unter Kontrolle ;)

**Comment by Stortebecker** [comment]
Jo, ich weiß, habe nur gemerkt, dass es noch nicht klappt. Aber da gebe ich Björn a bisserl Zeit...

**Comment by Stortebecker** [comment]
Comment to PeakPicking: On the test dataset used here, OpenMS HiResPeakPicker (1) indeed does a better job than vendor peak picking by msconvert (2). 2 is a subset of 1 and 1 has 17 more protein IDs.

**Comment by Stortebecker** [comment]
5. PeakPickerHiRes runs perfectly fine on single files, but not if the very same files are in a dataset collection.

**Comment by Stortebecker** [comment]
5. PeakPickerHiRes runs perfectly fine on single files, but not if the very same files are in a dataset collection.

**Comment by Stortebecker** [comment]
Record of the Gitter discussion on Peptide Search Engines:

- OpenMS wrappers exist for several search engines. However, they are hard to implement in Galaxy. We prefer in general to run the search engines on their own or out of SearchGUI. The wrappers will be made available, but not be used in tutorials.

- obsolete search engines: Sequest, [hm, den Rest finde ich nicht mehr]
- search engines to keep / make available: X!Tandem, Myrimatch, MSGF+, Comet (updated version of Sequest)

**Comment by Stortebecker** [comment]
PeakPickerHiRes crasht mal wieder mit:

> Warning: Cannot create folder '.OpenMS' in user home directory. Please check your environment! Home directory determined is: /usr/local/galaxy.

Das war doch schon behoben! Was ist da los?

**Comment by Stortebecker** [comment]
Ach, jetzt habe ich das PeakPicker-Problem gefunden. Manchmal denkt er, dass Profil-mzMLs schon zentriert sind und schluckt sie nicht. Aber das wird ein Tool-internes Problem sein, da kann man nix machen.
